demon.js

```diff
@@ -67,21 +93,6 @@ async function hasSuitableDate(data, xhrLink, secondShotXhrLink) {
       }
     }
 
-    if (data?.next_slot?.startsWith("2021-05")) {
```

Comment:
I think we can keep this, just without the `startsWith`

Reply:
👍